runtime/doc/various.txt

```diff
@@ -98,7 +98,9 @@ g8			Print the hex values of the bytes used in the
 
 						*:p* *:pr* *:print* *E749*
 :[range]p[rint] [flags]
-			Print [range] lines (default current line).
+			Print [range] lines (default current line). Can also
+			be spelled `:[range]|` due to Vi compatibility (see
+			|:bar|). Gives an error in an empty buffer.
```

Comment:
```suggestion
			Print [range] lines (default current line).  Can also
			be spelled `:[range]|` due to Vi compatibility (see
			|:bar|).  Gives an error in an empty buffer.
```

Reply:
Can you explain? Modern style is [a single space after terminating a punctuation](https://practicaltypography.com/one-space-between-sentences.html), and I'm not aware of a style guide for this prose that says otherwise… I hope I'm not missing something?

Reply:
It's been like this for a while. So let's try to stay consistent and use 2 spaces after punctuation. See also: https://english.stackexchange.com/questions/2544/how-many-spaces-should-come-after-a-period-full-stop I remember there were some discussions about it in one of the mailinglist, but I don't seem to find it.